daml-foundations/daml-tools/da-hs-daml-cli/tests/VisualTest.hs

```diff
@@ -0,0 +1,39 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+{-# LANGUAGE OverloadedStrings #-}
+module VisualTest
+   ( main
+   ) where
+
+import Test.Tasty
+import Test.Tasty.HUnit
+import System.Directory
+import System.IO.Extra
+import DA.Cli.Visual
+
+
+main :: IO ()
+main = defaultMain tests
+
+tests :: TestTree
+tests = testGroup "Tests" [unitTests]
+
+unitTests :: TestTree
+unitTests = testGroup "Unit tests"
+  [
+    testCase "file exists" (doesFileExist "daml-foundations/daml-tools/da-hs-daml-cli/visual-test-daml.dar" @? "missing dar file to process") ,
```

Comment:
I'm not sure this test and the one below are useful. I think there's not guarantee that the test are run in order anyway.

Reply:
Yeah, use a temp file for the test, and then you don't have the problem anyway

Reply:
These tests are not necessary. I added it in when I was messing around with bazel rules. Will get rid of them. If the files are not present the bazel command fails. So these tests are redundant.